qt/icons/src/CharIconPainter.cpp

```diff
@@ -12,6 +12,11 @@
 namespace MantidQt {
 namespace Icons {
 
+namespace {
+const QString defaultColor("black");
```

Comment:
Minor but code style says that these should be all caps with `_` separating words to indicate they are static.

Reply:
That too is a sound suggestion.